crates/core/c8y_api/src/smartrest/operations.rs

```diff
@@ -50,23 +52,31 @@ impl Operations {
         None
     }
 
-    pub fn filter_by_topic(&self, topic_name: &str) -> Vec<(String, Operation)> {
+    pub fn filter_by_topics(
```

Comment:
Originally, since `Operations` is a collection of custom operation handlers, the function is intended to extract the matched custom operation handlers by topic. However, it looks repurposed now.